modules/statics/src/networks.ts

```diff
@@ -401,15 +401,15 @@ class SolTestnet extends Testnet implements AccountNetwork {
 class Stx extends Mainnet implements StacksNetwork {
   name = 'Stx';
   family = CoinFamily.STX;
-  explorerUrl = 'https://explorer.stacks.co/';
+  explorerUrl = 'https://explorer.stacks.co/txid/';
   sendmanymemoContractAddress = 'SP3FBR2AGK5H9QBDH3EEN6DF8EK8JY7RX8QJ5SVTE';
   stakingContractAddress = 'SP000000000000000000002Q6VF78';
 }
 
 class StxTestnet extends Testnet implements StacksNetwork {
   name = 'StxTestnet';
   family = CoinFamily.STX;
-  explorerUrl = 'https://explorer.stacks.co/?chain=testnet';
+  explorerUrl = 'https://explorer.stacks.co/txid/?chain=testnet';
```

Comment:
I don't think this will work - the txid gets appended to the end of the URL so the query param would end up in the middle.

Reply:
Is there any other way to specify the chain other than a query param? Otherwise we might need to change how the `explorerUrl` works so that it's a format string instead of a string to append to.

Reply:
We had to deal with the same issue in Solana https://github.com/BitGo/BitGoJS/blob/master/modules/statics/src/networks.ts#L398
But its the UI doing the split at `?`
https://github.com/BitGo/lumina/blob/master/packages/shared/src/api/utils.ts#L307-L311

Reply:
Got it, thanks